From 9083af7c9b5b0ccf65e8d64b3be21b3c8100a70d Mon Sep 17 00:00:00 2001
From: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
Date: Mon, 4 Nov 2013 15:23:06 +0100
Subject: [PATCH] xor: set xor min count

If iperf is started in server mode and a TCP test is performed the xor/dma
engine fails.
According to the 88F6282 datasheet the minimum byte count for XOR engine is
16 Bytes. Set the MV_XOR_MIN_BYTE_COUNT of 128 to 16.

Signed-off-by: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
---
 drivers/dma/mv_xor.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/mv_xor.h b/drivers/dma/mv_xor.h
index c632a47..3bd5286 100644
--- a/drivers/dma/mv_xor.h
+++ b/drivers/dma/mv_xor.h
@@ -163,7 +163,7 @@ struct mv_xor_desc {
 #define mv_hw_desc_slot_idx(hw_desc, idx)	\
 	((void *)(((unsigned long)hw_desc) + ((idx) << 5)))
 
-#define MV_XOR_MIN_BYTE_COUNT	(128)
+#define MV_XOR_MIN_BYTE_COUNT	(16)
 #define XOR_MAX_BYTE_COUNT	((16 * 1024 * 1024) - 1)
 #define MV_XOR_MAX_BYTE_COUNT	XOR_MAX_BYTE_COUNT
 
-- 
1.7.9.5

